### PR TITLE
travis jobs: enable Windows

### PR DESCRIPTION
**Pull request by joshmoore**
Follow on to #132. Run Windows in Travis.

### CODE REVIEW

**Comment by joshmoore** [comment]
All platforms are running (periodically with tables failures) and a real issue with the Windows test arose:

```
>           return open(string, self._mode)
E           FileNotFoundError: [Errno 2] No such file or directory: 'C:UserstravisAppDataLocalTemppytest-of-unknownpytest-0popen-gw3testLoad0test'
```

**Comment by joshmoore** [comment]
Green!

File: .travis.yml [hunk 1]
```diff
@@ -1,41 +1,48 @@
...
+        - tox -e py37
+    - name: Python36
+      os: trusty
+      python: "3.6"
+      script:
+        - tox -e py36
+    - stage: deploy
+      os: trusty
+      python: "3.6"
+      script:
```
Comment (manics) [inline .travis.yml]: Can you remove this `script` section? It wasn't in the original deploy step.

Comment (joshmoore) [reply .travis.yml]: Isn't that the equivalent of the `$TOX = no` clause?

Comment (manics) [reply .travis.yml]: `skip_cleanup` wasn't set, so it should've been deployed from a clean environment: https://docs.travis-ci.com/user/deployment#uploading-files-and-skip_cleanup

Comment (joshmoore) [reply .travis.yml]: Hmmm.... I'm not following, but it looks like all the rst-lint actions are now in tox anyway.

**Comment by joshmoore** [comment]
Few commits pushed, @manics.

**Comment by manics** [comment]
Might need `script: skip` too

**Comment by joshmoore** [comment]
That's us back green again, @manics. Not sure why both skips are needed.

**Comment by joshmoore** [comment]
` ¯\_(ツ)_/¯` Note: this may have further issues when it comes time to deploy a tag, but I'll deal with that during release.

**Pull request merged**